extensions/wikia/WikiFactory/Hubs/WikiFactoryHub.php

```diff
@@ -343,11 +310,7 @@ public function getCategoryByName( $name ) {
 
 	private function loadCategories() {
 
-		global $wgExternalSharedDB;
-		if( !$wgExternalSharedDB ) {
-			return array();
-		}
-
+		// TODO: add caching
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this "TODO" comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)

[project-id]: wikia:app